index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="utf-8" />
+  <title>課題</title>
+  <meta name="description" content="ページの解説" />
+</head>
+
+<body>
+  <header class="header">
+  <!-- /.header --></header>
+
+  <h2>FizzBuzz問題</h2>
+    <form action="" method="post" id="form1">
+      <label>FizzNum</label>
+      <input id="fizznum" type="number" name="FizzNum"><br>
+      <label>BuzzNum</label>
+      <input id="buzznum" type="number" name="BuzzNum"><br>
+      <input type="button" name="exec" value=実行 onclick="javascript:submitForm(); return false;" />
+      <!-- <button type="submit">実行</button> -->
+    </form>
+    <div id="div1">【出力】</div>
+</body>
+
+<script>
+  var form1 = document.getElementById('form1');
```

Comment:
`var` は昨今ほとんど使われません。
再代入しないなら `const` 再代入をする場合は `let` を利用しましょう。

```suggestion
  const form1 = document.getElementById('form1');
```